src/Centreon/Domain/Monitoring/Exception/ResourceException.php

```diff
@@ -22,6 +22,6 @@
 
 namespace Centreon\Domain\Monitoring\Exception;
 
-class ResourceRegExpException extends \PDOException
+class ResourceException extends \Exception
```

Comment:
what is the idea of this exception class if it's replaced by `RequestParametersTranslatorException`?

Reply:
The idea is to use a context exception while keeping track of the original exception. The current choice and not to let technical problems show up in the error message. A problem in a repostiory is a technical problem.